src/app/document.js

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
```

Comment:
This Document should be moved to `src/app/containers/Document` and be a `jsx` file.

Reply:
👍